Forms.html

```diff
@@ -0,0 +1,85 @@
+    <!DOCTYPE html>
+    <html>
+        <head>
+    <title>Forms</title>
+    <meta charset="UTF-8">
+        </head>
+        <body>
+    <form>
+        <fieldset>
+            <legend>Basic User Information:</legend>
+        <label for="fname">Full name:</label>
+        <input type="text" id="fname" name="fname" required><br>
+        <label for="email">Email:</label>
+        <input type="email" id="email" name="email" required><br>
+        <label for="pwd">Password:</label>
+        <input type="password" id="pwd" name="pwd" required>
+        <label for="cpwd">Confirm Password:</label>
+        <input type="password" id="cpwd" name="cpwd" required><br>
+        <label >Gender:</label>
+        <input type="radio" id="male" name="Gen" value="Male">
+        <label for="male">Male</label>
+        <input type="radio" id="female" name="Gen" value="Female">
+        <label for="female">Female</label>
+        <input type="radio" id="other" name="Gen" value="Other">
+        <label for="other">Other</label><br>
+        <label for="DOB">Date of Birth:</label>
+        <input type="date" id="DOB" name="DOB">
+    </fieldset>
+    <fieldset>
+        <legend>Address Information:</legend>
+    <label for="Address">Street Address:</label>
+    <textarea id="Address" name="Address"></textarea><br>
+    <label for="country">Country:</label>
+    <select id="country" name="country">
+    <option value="USA">USA</option>
+    <option value="Scotland">Scotland</option>
+    <option value="Moldova">Moldova</option>
+    <option value="Ucraine">Ucraine</option>
+    <option value="Sweeden">Sweeden</option>
+    </select><br>
+    <label for="city">City:</label>
+    <input type="text" id="city" name="city" > <br>
+    <label for="pc">Postal Code:</label>
+    <input type="number" id="pc" name="pc">
+    </fieldset>
+    <fieldset>
+        <legend>Preferences</legend>
+        <label>Preferred Contact Method</label>
+        <input type="checkbox" id="e-mail" name="pcm" value="Email">
+        <label for="e-mail">Email</label>
+        <input type="checkbox" id="phone" name="pcm" value="Phone">
+        <label for="phone">Phone</label>
+        <input type="checkbox" id="posmail" name="pcm" value="Postal-Mail">
+        <label for="posmail">Postal Mail</label><br>
+        <label for="color">Favorite color:</label>
+        <input type="color" id="color" name="color">
+    </fieldset>
+    <fieldset>
+        <legend>Additional Details:</legend>
+        <label for="hdha">How did you hear about us?</label>
+        <select id="hdha" name="hdha">
+            <option value="internet">Internet</option>
+            <option value="friend">Friend</option>
+            <option value="adv">Advertisement</option>
```

Comment:
c'mon, don't be lazy.
Write the word in full. Imagine that on backed will work somebody else and he doesn't understand what you mean by "adv"